components/kubeflow/dnntrainer/build_image.sh

```diff
@@ -19,7 +19,7 @@ while getopts ":hp:t:i:b:l:" opt; do
         echo "-t: tag name"
         echo "-i: image name. If provided, project name and tag name are not necessary"
         echo "-b: tensorflow base image tag. Optional. The value can be tags listed under \
-        https://hub.docker.com/r/tensorflow/tensorflow/tags. Defaults to '1.6.0'."
+        https://hub.docker.com/r/tensorflow/tensorflow/tags. Defaults to '2.3.0'."
```

Comment:
nit:
`0.23.0`?

Reply:
I think `2.3.0` is correct

Reply:
tensorflow is 0.23.0, while tensorflow transform and tensorflow model analysis is 2.3.0